tests/conftest.py

```diff
@@ -25,6 +32,18 @@ def pytest_collection_modifyitems(config, items):
         if "slow" in item.keywords:
             item.add_marker(skip_slow)
 
+    # Limit the number of permutations of parametrised tests to run.
+    limit = config.getoption("--limit")
+    if limit > 0:
+        tests_by_name = {item.name: item for item in items}
+        # Add the name of parametrize base tests to this list.
+        tests_to_skip_parametrize = ["test_noise_order"]
```

Comment:
This is where we can add tests to only run the specific number of parametrized args.